react/week1/todos/src/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+// import App from './App';
+// import * as serviceWorker from './serviceWorker';
+
+
+const root = document.getElementById('root')
+
+ function TodoItem(props) {
+   return <li>{props.description}, {props.deadline}</li>
+ }
+
+class Todo extends React.Component {
+  render() {
+   const todoList = this.props.todos;
+   const result = todoList.map(element => {
+    return <TodoItem description={element.description} deadline={element.deadline}></TodoItem>
+  })
+   return (
+     <ul>{result}</ul>
+   )
+  }
+}
+
+const todosArray = [
+  { description: 'Watch TV', deadline: new Date('2020-04-24').toDateString()},
+  { description: 'Listen music', deadline: new Date('2020-02-20').toDateString()},
+  { description: 'Wash clothes', deadline: new Date('2020-03-28').toDateString()}
+]
+
+class Header extends React.Component {
```

Comment:
Nothing is wrong here, but simple components such as this one can be written just as plain functions:

```jsx
function Header() {
 return (<header>
      <h3>Todo list</h3>
    </header>))
}
```

Less verbosity ;)